benchmarks/src/jmh/java/org/postgresql/util/ConnectionUtil.java

```diff
@@ -62,7 +62,7 @@ public static Properties getProperties() {
 
     PGProperty.USER.set(properties, getUser());
     PGProperty.PASSWORD.set(properties, getPassword());
-    PGProperty.PG_PORT.set(properties, getPort());
+    PGProperty.PORT.set(properties, getPort());
```

Comment:
This would be quite invasive. If we want to change to PORT then leave PG_PORT in and deprecate it. Other than preference is there a reason to change this ?